common/src/java/org/apache/hadoop/hive/ql/ErrorMsg.java

```diff
@@ -529,6 +529,7 @@ public enum ErrorMsg {
   PARQUET_FOOTER_ERROR(20022, "Failed to read parquet footer:"),
   PARQUET_UNHANDLED_TYPE(20023, "Unhandled type {0}", true),
   ORC_FOOTER_ERROR(20024, "Failed to read orc footer:"),
+  NOT_LEAP_YEAR(20025, "not a leap year", true),
```

Comment:
I have the impression that this class holds messages that are targeting the end-user. I am not sure if it's appropriate for the use case here since we are not throwing this error anywhere.

Reply:
moved the message out of ErrorMsg. Fixed.